base/shell/cmd/dir.c

```diff
@@ -1782,6 +1782,10 @@ ResolvePattern(
             pszPatternPart = NULL;
         }
     }
+    else if (_istalpha(pNextDir[0]) && pNextDir[1] == TEXT(':') && pNextDir[2] == 0)
+    {
+        /* Drive name only */
+    }
```

Comment:
Is it intentional that there's no code in this condition body?

Reply:
Yes, in this case, it doesn't execute the below code.

Reply:
Suggestions:
`pNextDir[2] == TEXT('\0')`
`/* Drive name only: nothing to do. */`

Reply:
OK.